src/pyfuse3.pyx

```diff
@@ -393,6 +393,37 @@ cdef class FileInfo:
             out.nonseekable = 0
 
 
+@cython.freelist(1)
+cdef class ConnInfo:
+    '''
+    Instances of this class store information about the fuse connection.
+    The attributes correspond to the elements of the ``fuse_conn_info`` struct.
+    '''
+
+    cdef fuse_conn_info conn
```

Comment:
What's the point of keeping a copy of the whole struct around, if all you care about is the `max_read` parameter?